test_fms/data_override/data_table_base

```diff
@@ -2,3 +2,6 @@
 "ICE", "sic_obs",  "SIC", "INPUT/sst_ice_clim.nc", .false., 300.0
 "OCN", "sst_obs",  "SST", "INPUT/sst_ice_clim.nc", .false., 300.0
 "LND", "sst_obs",  "SST", "INPUT/sst_ice_clim.nc", .false., 300.0
+"ATM", "so2_cont_volc", "SO2_cont_volc", "INPUT/so2_with_coeffs.nc", .false., 1.0
+"ATM", "so2_cont_volc_coeff", "SO2_cont_volc_coeff", "INPUT/so2_with_coeffs.nc", .false., 1.0
+"ATM", "so2_cont_volc_mod_coeff", "", "", .false., 0.8
```

Comment:
Are these updates required for the 3D implementation and data nudge procedure?  These additions seem like they should be mentioned separately in a commit message.

Reply:
They are required for the 3D / nudge test (can/will separate out into different commit if desired).